src/controllers/facilitiesFeedbackController.js

```diff
@@ -0,0 +1,158 @@
+import model from '../database/models';
+import decodeToken from '../utils/decodeToken';
+import validateFeedback from '../middlewares/feedbackValidator';
+
+exports.sendFacilityFeedback = async(req,res) => {
+    const {error, errorMessage} = await validateFeedback(req.body);
+    if(error) return res.status(400).json({message: res.__(errorMessage)})
+    const currentUser = await decodeToken(req,res);
+    const {facility,facilityNotFound} = await findFacilityById(req.params.facilityId);
+    if(facilityNotFound == true)
+      return res.status(404).json({success: false, message: res.__("Facility not found")});
+     const newFeedback = {
+      facilityId:req.params.facilityId, senderId:currentUser.id, 
+      feedback_title: req.body.feedback_title, feedback_content: req.body.feedback_content, seen: false
+    }
+    let feedback;
+    try{
+        feedback = await  model.FacilitiesFeedback.create(newFeedback);
+    }catch(e){
+     return   res.status(500).json({sucess: false, message:res.___("Error")});
+    }
+    return res.status(201).json({
+        success: true,
+        message: res.__("Feedback sent successfully"),
+        feedback,
+        sender: currentUser,
+        facility
+    })
+}
+
+exports.getByFeedbackId = async(req,res) => {
+    const {feedback,feedbackNotFound} = await findFeedbackById(req.params.feedbackId);
+    if(feedbackNotFound)
+      return res.status(404).json({
+         success: false,
+         message: res.__("Feedback not found")
+      })
+    const {user, userNotFound} = await findUserById(feedback.dataValues.senderId);
+     if(userNotFound)
+      return res.status(400).json({message: res.__("Feedback sender not found")})
+     const {facility, notFound} = await findFacilityById(feedback.dataValues.facilityId);
+     if(notFound)  return res.status(400).json({message: res.__("Facility  not found")})
+
+     return res.status(200).json({
+         success:true,
+         feedback: {data: feedback,facility: facility.dataValues,sender: user.dataValues
+       }
+     })
+}
+
+exports.getAllFeedbacks = async(req,res) => {
+    const feedbacks = await model.FacilitiesFeedback.findAll();
+    return res.status(200).json({feedbacks});
+}
```

Comment:
I feel like this endpoint is useless, because it doesn't make sense having all feedbacks separately from facility.... Getting all feedbacks for a certain facility is enough and it makes sense. Even though we have feedbacks separately but are associated to facility because you can't have feedbacks without facility, therefore you should get feedbacks of a certain facility.

At least explain this..